### PR TITLE
Mt3 638

### PR DESCRIPTION
**Pull request by LBHBomotoso**

### CODE REVIEW

File: src/Components/ETRA/EmailSignOff/index.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  }
+
+  handleReviewLater = async () => { 
+    this.setState({ isAttemptingToSave: true });
+    /*const successful = await this.createMeeting.Execute(this.getUnreviewedMeetingModel());
+
+    if (successful) {
+      this.props.onReviewLater();
+    }
+    else {
```
Comment (LBHABEBIC) [inline src/Components/ETRA/EmailSignOff/index.tsx:98]: maybe remove all this commented code if not needed anymore?

File: src/Components/ETRA/SignatureSignOff/index.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    if(this.props.signOffMode){
+      successful = await this.signoffMeeting.Execute(this.getMeetingSignOffModel());
+    }else{
+      successful = await this.createMeeting.Execute(this.getMeetingModelWithSignatureTrimmed());
+    }
+
+    if (successful) {
+      //this.props.onReviewNow();
+    }
+
```
Comment (LBHABEBIC) [inline src/Components/ETRA/SignatureSignOff/index.tsx:122]: should this have something in the if statement?

File: src/Components/ETRA/EmailSignOff/index.tsx (hunk 1 above)
Comment (LBHBomotoso) [reply src/Components/ETRA/EmailSignOff/index.tsx:98]: I will do. I'm just getting an understanding of how the code works. 
It's easier for me to uncomment stuff if I think I need it.

File: src/Components/ETRA/SignatureSignOff/index.tsx (hunk 2 above)
Comment (LBHBomotoso) [reply src/Components/ETRA/SignatureSignOff/index.tsx:122]: See previous comment. I will clean these up when the application is complete

**Pull request merged**